core/audits/audit.js

```diff
@@ -124,9 +124,10 @@ class Audit {
    * @param {LH.Audit.Details.Table['headings']} headings
    * @param {LH.Audit.Details.Table['items']} results
    * @param {LH.Audit.Details.Table['summary']=} summary
+   * @param {LH.Audit.Details.Table['sortBy']=} sortBy
    * @return {LH.Audit.Details.Table}
    */
-  static makeTableDetails(headings, results, summary) {
+  static makeTableDetails(headings, results, summary, sortBy) {
```

Comment:
The way it is now, it seems like this function will actively sort the table when you call it which isn't the case. Could we rename this to `sortedBy`.